scripts/upgrade

```diff
@@ -79,56 +93,84 @@ ynh_add_nginx_config
 ynh_script_progression --message="Upgrading dependencies..." --weight=1
 
 ynh_install_app_dependencies $pkg_dependencies
+ynh_install_python --python_version=$python_version
```

Comment:
It's enough to upgrade python ?

Is there a risk that a new python version with hotfix appears and this package was not updated quickly (compared to debian package) ?

Reply:
> 
> 
> It's enough to upgrade python ?

Yes it's enough `ynh_install_python`: 

1. install or update pyenv
2. install requested `$python_version`
3. recreate an `$app` virtualvenv based on `$python_version`
4. cleanup no more needed python version

> 
> Is there a risk that a new python version with hotfix appears and this package was not updated quickly (compared to debian package) ?

For now, Debian 10 is provided with:
- Python 2.7.16 (March 4, 2019) : latest official is 2.7.17 (Oct. 19, 2019)
- Python 3.7.3 ( March 25, 2019) : latest is 3.7.10 (Feb. 15, 2021)
- There is also 3.8.8 (Feb. 19, 2021) and 3.9.2 (Feb. 19, 2021)

So, I'm pretty sure that yunohost python will be up to date ^^ not 2 years old....

Reply:
To me it ain't clear what's the benefit of installing a specific version of Python here ... I mean Buster already ships Python 
3.7.3 (= `$python_version` ?). Python is supposed to be forward-compatible (with the exception of Python 2 vs. 3 of course) sooooooo ...?

I mean from what I know of pepettes it should be super-lightweight but compiling Python locally adds a huge cost ? :s

Reply:
(Source: i'm typing this while waiting for the job on ci-apps-dev to finish while it compiles python for the 10th time :/ ...)

Reply:
I fully agree that for pepettes, having `ynh_install_python` is overkilled... It's more a proof of concept.. As pepettes is small, it's quicker to test on it.

Reply:
ynh_install_python removed